src/WebJobs.Extensions/Extensions/Timers/Listener/TimerListener.cs

```diff
@@ -112,7 +112,7 @@ internal System.Timers.Timer Timer
 
             // log the next several occurrences to console for visibility
             string nextOccurrences = TimerInfo.FormatNextOccurrences(_schedule, 5);
-            _trace.Verbose(nextOccurrences);
+            _trace.Info(nextOccurrences, "WebJobs.Execution");
```

Comment:
New commit now has the latest from Core. I verified the schedule shows up with Info now.